src/test/java/org/broadinstitute/hellbender/tools/walkers/genotyper/GenotypeLikelihoodCalculatorsUnitTest.java

```diff
@@ -14,7 +14,7 @@
 
     GenotypeLikelihoodCalculators calcs;
 
-    @BeforeSuite
+    @BeforeClass
     public void init(){
```

Comment:
I feel like most of these before suite should just be initialized inline where they're declared.  Or replaced with a getter that returns a new instance of the thing on demand.  That's a bigger change though.

Reply:
Agreed. I was trying to minimize the changes, but wanted to get rid of unnecessary @BeforeSuites.